### PR TITLE
Update REadme for us on fork

### PR DESCRIPTION
**Pull request by dstull-github**
test fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,12 @@
...
-5. bundle install
 6. rake db:setup
 7. start rails server: `rails s`
 8. navigate to http://localhost:3000/servers
 
 # Testing
-1. inside app directory, run: `rake`
\ No newline at end of file
+1. inside app directory, run: `rake`
+
```
Comment (dstull-github) [inline README.md:20]: What about this one

File: README.md [hunk 2]
```diff
@@ -11,10 +11,12 @@
 2. install ruby 2.6.2
 3. install bundler gem
 4. go inside app directory
-5. bundle install
```
Comment (dstull-github) [inline README.md:14]: And this one

File: README.md [hunk 3]
```diff
@@ -11,10 +11,12 @@
...
 6. rake db:setup
 7. start rails server: `rails s`
 8. navigate to http://localhost:3000/servers
 
 # Testing
-1. inside app directory, run: `rake`
\ No newline at end of file
+1. inside app directory, run: `rake`
```
Comment (dstull-github) [inline README.md:19]: And here

File: README.md [hunk 4]
```diff
@@ -11,10 +11,12 @@
...
+
+# Do new things!
+1. and more
```
Comment (dstull) [inline README.md:22]: What about this one from the owner

**Pull request closed without merging**